content/en/blog/2025/kubeletstats-receiver-metrics-deprecation.md

```diff
@@ -0,0 +1,148 @@
+---
+title:
+  'Kubernetes CPU Metrics in the kubeletstats Receiver: Transition from
+  .cpu.utilization to .cpu.usage'
+linkTitle: kubeletstats Receiver metrics update
+author: >
+  [Christos Markou](https://github.com/ChrsMark) (Elastic), [Tyler
+  Helmuth](https://github.com/TylerHelmuth) (Honeycomb), [Dmitrii
+  Anoshin](https://github.com/dmitryax) (Cisco/Splunk)
+date: 2025-05-26
+issue: https://github.com/open-telemetry/opentelemetry.io/issues/6847
+cSpell:ignore: Anoshin Dmitrii Helmuth Kubelet kubeletstats Kubelet’s Markou
+---
+
+## Kubernetes CPU Metrics in the `kubeletstats` Receiver: Transition from `.cpu.utilization` to `.cpu.usage`
```

Comment:
No need to repeat the title here. Hugo automatically adds the front matter title as the page title:

<img width="762" alt="Screenshot 2025-05-29 at 3 58 39 PM" src="https://github.com/user-attachments/assets/3080c5e9-8de0-40f2-9649-b2921ace8b47" />

```suggestion

```